core/trino-main/src/test/java/io/trino/operator/join/unspilled/TestHashJoinOperator.java

```diff
@@ -1314,6 +1313,34 @@ public static Object[][] testMemoryLimitProvider()
                 {false, false}};
     }
 
+    @DataProvider(name = "singleBigintLookupSourceProvider")
+    public static Object[][] singleBigintLookupSourceProvider()
+    {
+        return new Object[][] {{true}, {false}};
+    }
+
+    @DataProvider(name = "hashJoinTestValuesAndsingleBigintLookupSourceProvider")
+    public static Object[][] hashJoinTestValuesAndsingleBigintLookupSourceProvider()
```

Comment:
nit: consider writing cross product method for combining these. I'm pretty sure we have something like that in code